conftest.py

```diff
@@ -161,3 +162,16 @@ def enable_mlflow_testing():
     with pytest.MonkeyPatch.context() as mp:
         mp.setenv(_MLFLOW_TESTING.name, "TRUE")
         yield
+
+
+@pytest.fixture(scope="module", autouse=True)
+def disable_get_bucket_location(request):
```

Comment:
Why do we need this fixture?

Reply:
This is because we now get the region name of the bucket upon the initialization of the `S3ArtifactRepository`, which breaks several tests. Perhaps, I could only get the region name when `optimized`? That would prevent the need for this fixture.

Reply:
You know what, let me just remove this fixture and only get the region name for now when `optimized`. It preserves more of the old behavior.